.github/workflows/maven-release-perform.yml

```diff
@@ -16,22 +16,28 @@ concurrency:
 
 jobs:
   perform:
-    name: Release perform {{ inputs.version }}
+    name: Release perform ${{ inputs.version }}
     runs-on: ubuntu-latest
     steps:
       - name: Checkout the latest code
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
+          ref: master
       - name: Setup local maven cache
         uses: actions/cache@v2
         with:
           path: ~/.m2/repository
           key: checkstyle-maven-cache-${{ hashFiles('**/pom.xml') }}
-      - name: Set up JDK 11
-        uses: actions/setup-java@v3
-        with:
-          java-version: 11
+      - name: Configure GPG
+        env:
+          GPG_SECRET_KEY: ${{ secrets.GPG_SECRET_KEY }}
+        run: |
+          mkdir -p ~/.gnupg/
+          echo "$GPG_SECRET_KEY" > ~/.gnupg/private.key
+          gpg --batch --import ~/.gnupg/private.key
+          chmod 600 ~/.gnupg/private.key
+          chmod 700 ~/.gnupg
```

Comment:
These two are required because of ` gpg: WARNING: unsafe permissions on homedir`
 https://github.com/stoyanK7/checkstyle/actions/runs/4204292665/jobs/7294759223#step:6:1802

Fix from:  https://gist.github.com/oseme-techguy/bae2e309c084d93b75a9b25f49718f85